<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1291</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>10</y>
      <width>631</width>
      <height>521</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>490</y>
      <width>631</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="clearButton">
       <property name="text">
        <string>Clear Scene</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="drawButton">
       <property name="text">
        <string>Draw Route i</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="iteration"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>10</y>
      <width>631</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="generateButton">
       <property name="text">
        <string>Generate</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="algoBox">
       <item>
        <property name="text">
         <string>AntAlgo</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>FiringAlgo</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="dotNumber"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>631</width>
      <height>291</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>                          AntAlgo</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>alpha</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="antAlpha"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>beta</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="antBeta"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>startFer</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="antStartFer"/>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>iterationNumber</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLineEdit" name="antIterationNumber"/>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="antFerK"/>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>ferK</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>              FiringAlgo</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QFormLayout" name="formLayout_2">
         <item row="0" column="1">
          <widget class="QLineEdit" name="FireMinT"/>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>minT</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>maxT</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>iterationNumber</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="FireMaxT"/>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="fireIterationNumber"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1291</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
